### PR TITLE
Add `krusader, plex-update-tool, plex-meta-manager` to addons. Add `webtop` to selfhosted. Added more to environment.

### PR DESCRIPTION
**Pull request by aelfa**

### CODE REVIEW

**Review by doob187 — CHANGES_REQUESTED** [review]
Add environments also to 

https://github.com/dockserver/dockserver/blob/master/traefik/templates/compose/.env

**Comment by aelfa** [comment]
@doob187 On it now!

**Comment by aelfa** [comment]
@doob187 Done! Kindly review.

File: apps/addons/krusader.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  krusader:
+    hostname: 'krusader'
+    container_name: 'krusader'
+    environment:
+      - 'PGID=${ID}'
+      - 'PUID=${ID}'
+      - 'TZ=${TZ}'
+      - 'UMASK=${UMASK}'
+      - 'VNC_PASSWORD=${KRUSADERPASSWORD}'
+      - 'WEBPAGE_TITLE=Tower'
```
Comment (doob187) [inline apps/addons/krusader.yml]: Environment pls

**Review by doob187 — CHANGES_REQUESTED** [review]
Read notes

File: apps/addons/plex-update-tool.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+services:
+  plex-update-tool:
+    hostname: 'plex-update-tool'
+    container_name: 'plex-update-tool'
+    environment:
+      - 'PGID=${ID}'
+      - 'PUID=${ID}'
+      - 'TZ=${TZ}'
+      - 'TMDB_API_KEY=${TMDBAPI}'
+      - 'RUN_EVERY_N_HOURS=12'
```
Comment (doob187) [inline apps/addons/plex-update-tool.yml]: Variable  plz

Made customizable possible

**Review by doob187 — COMMENTED** [review]
One changes plz then 

👍

**Review by doob187 — APPROVED** [review]

**Pull request merged**